api/src/main/java/io/cloudevents/format/json/CloudEventSerializer.java

```diff
@@ -0,0 +1,23 @@
+package io.cloudevents.format.json;
+
+import com.fasterxml.jackson.core.JsonGenerator;
```

Comment:
What do you think about move Jackson dependencies to json-jackson module? Check out this issue #113

Reply:
Without any external dependency in the API module, anyone who wants to use the API is free of conflicts with pre-existing frameworks and libraries.

Reply:
also see https://github.com/cloudevents/sdk-java/pull/108#discussion_r411153130